src/file_readers/common/sql_reader/tables/pasef_frame_msms.rs

```diff
@@ -28,6 +29,7 @@ impl ReadableFromSql for PasefFrameMsMsTable {
                 .read_column_from_table("CollisionEnergy", table_name),
             precursor: sql_reader
                 .read_column_from_table("Precursor", table_name),
+            collision_energy_by_precursor: sql_reader.get_data_from_sql(&"select CollisionEnergy from PasefFrameMsMsInfo group by Precursor".to_string()),
```

Comment:
I think this conflicts with the setup of this class. It should just use the `read_from_table` structure to ensure the length of the array and the order is consistent with the rest of the table. Parsing it for a specific use case should happen at the appropriate spot, not here.